tests/test_structured_state.py

```diff
@@ -531,15 +530,15 @@ def test_circ_approx_explicit_ttn(circuit: Circuit) -> None:
         # Check for TTNxGate
         cfg = Config(truncation_fidelity=0.99, leaf_size=3, float_precision=np.float32)
         ttn_gate = simulate(libhandle, circuit, SimulationAlgorithm.TTNxGate, cfg)
-        assert np.isclose(ttn_gate.get_fidelity(), 0.769, atol=1e-3)
+        assert np.isclose(ttn_gate.get_fidelity(), 0.751, atol=1e-3)

```

Comment:
Since NetworkX provides a worse partition than KaHyPar, the fidelity in the simulation drops by a bit.

Reply:
Are you sure that you want to update the expected value and not the tolerance?

Reply:
Hmm, I've been using these tests for a very rudimentary regression/improvement tracking, which is why they have a very particular value for the fidelity. It has happened before that changes that I did not expect that would change the fidelity did. If I made the tolerance margin larger, it's likely some of these wouldn't be caught.